student.rb

```diff
@@ -1,8 +1,10 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(classroom:, name: 'Unknown', age: 0, parent_permission: true)
-    super(name: name, age: age, parent_permission: parent_permission)
+  attr_reader :classroom
+
+  def initialize(classroom:, name: 'Unknown', age: 0)
+    super(name, age)
     @classroom = classroom
   end
```

Comment:
- [ ] [REQUIRED] Good job so far! When setting the classroom for a `student` it also adds it to the classrooms' students:

```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```